doc/ref/configuration/master.rst

```diff
@@ -193,6 +193,50 @@ following the Filesystem Hierarchy Standard (FHS) might set it to
 
     pki_dir: /etc/salt/pki/master
 
+
+.. conf_master:: cluster_id
+
+``cluster_id``
+--------------
+
+.. versionadded:: 3007
+
+When defined the master will operate in cluster mode. The master will send the cluster key and id to minions instead of it's own key and id. The master will also forward it's local event bus to other master defined by ``cluster_peers``
```

Comment:
1. When defined`,` the master will...
2. ...other master`s` defined by \`\`cluster_peers\`\`